src/styles/blocks/container.scss

```
@@ -0,0 +1,58 @@
+.container {
+  width: #{$container-size};
+  margin: auto;
+  padding: 50px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap-size};
+
+  &__cell {
+    box-sizing: border-box;
+    background-color: #{$main-color};
+    border: 1px solid #{$border-color};
+  
+    font-family: "Arial", sans-serif;
+    font-size: 30px;
+    line-height: #{$cell-size};
+    text-align: center;
+
+    transition: transform 0.5s, background-color 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffc0cb;
+      transform: translateY(-20px);
+    }    
+  
+    @include square($cell-size);
+   
+    @for $index from 1 through 31 {
+      &:nth-of-type(#{$index}):before {
+      content: "#{$index}";
+      }
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length--#{$index}
+    &__cell:nth-child(n + #{$index + 1}) {
+      display: none;
+    }
+  }
+
+  $days-of-week:
+  "mon" 0,
+  "tue" ($cell-size + $gap-size),
+  "wed" (($cell-size + $gap-size) * 2),
+  "thu" (($cell-size + $gap-size) * 3),
```

don't repeat your calculation everywhere, just save the corresponding number and use it in your loop to do the calculation
```suggestion
  "tue" 1,
  "wed" 2,
  "thu" 3,
```